webapp/components/_new/features/PageParamsLink/PageParamsLink.vue

```diff
@@ -1,15 +1,19 @@
 <template>
   <nuxt-link
-    v-if="pageParams.isInternalPage"
+    v-if="!forceTargetBlank && pageParams.isInternalPage"
     :to="pageParams.internalPage.pageRoute"
     :data-test="pageParams.name + '-nuxt-link'"
   >
     <slot />
   </nuxt-link>
   <a
     v-else
-    :href="pageParams.externalLink.url"
-    :target="pageParams.externalLink.target"
+    :href="
+      pageParams.isInternalPage ? pageParams.internalPage.pageRoute : pageParams.externalLink.url
+    "
+    :target="
+      forceTargetBlank ? '_blank' : pageParams.isInternalPage ? '' : pageParams.externalLink.target
+    "
```

Comment:
I find it difficult to understand this and the whole component!

I would expect a component to be pretty blunt but now you set some logic in here, 
I think this complexity makes the reusability of the component pretty hard.

Reply:
I would prefer to give the href & the target in the component as props and let the logic what is given to the props be done when calling it! (ex. webapp/components/Registration/RegistrationSlideCreate.vue)

Reply:
pageParams would be a to wide definition.

Reply:
The component is used without logic all over the place, because it's always the same except in the new case.

With the name you are kind of right, but at that time I couldn't find an other short fitting name.

Please lets talk about it.